version.py

```diff
@@ -0,0 +1,5 @@
+if __name__ == '__main__':
```

Comment:
Why add this file?

Reply:
To be able to get the version of a package via a standard way, running this version.py in the root of the package. It can be used by other tools to check the version (even of a non-installed package) or get the version automatically when releasing to master.